app/index.jsx

```diff
@@ -151,7 +164,8 @@ class App extends Component {
             window.location.href = `${origin}${pathname}?${url.toString()}`;
         }
         if (settings.historical) {
-            this.handleDateChange(today);
+            this.cleanComparison();
+            window.location.reload(false);
```

Comment:
while user working with the chart, we should remove added `comparisons`, so we just try to update local storage and then reload the page so that chart load these data again.

Reply:
yes, it makes sense to refresh the app - since there's no way to programmatically set the comparisons.